### PR TITLE
feat: add AssetInput loading animation

### PR DESCRIPTION
**Pull request by NadiaPosch**
Concerns this ticket: https://app.clickup.com/t/23ypf0v

Adds a loading prop and animation for the AssetInput

### CODE REVIEW

File: src/components/AssetInput/AssetInput.tsx [hunk 1]
```diff
@@ -82,9 +84,14 @@ export type AssetInputProps =
           size?: undefined;
           onUploadClick?: () => void;
           onLibraryClick?: () => void;
+          isLoading?: boolean;
       };
 
-const AssetThumbnail: FC<Pick<AssetProps, "asset" | "size"> & { isActive?: boolean }> = ({ asset, size, isActive }) => (
+const AssetThumbnail: FC<Required<Pick<AssetProps, "asset" | "size">> & { isActive?: boolean }> = ({
```
Comment (SamuelAlev) [inline src/components/AssetInput/AssetInput.tsx:90]: The type is becoming more complex => make it simpler and extract to own type?
```ts
type AssetThumbnailProps = {
  asset: Required<AssetProps['asset']>,
  size: AssetProps['size'],
  isActive?: boolean
}
```

File: src/components/AssetInput/SpinningCircle.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { merge } from "@utilities/merge";
```
Comment (SamuelAlev) [inline src/components/AssetInput/SpinningCircle.tsx:1]: Missing header

File: src/components/AssetInput/AssetInput.tsx (hunk 1 above)
Comment (NadiaPosch) [reply src/components/AssetInput/AssetInput.tsx]: Done 👍

File: src/components/AssetInput/SpinningCircle.tsx (hunk 2 above)
Comment (NadiaPosch) [reply src/components/AssetInput/SpinningCircle.tsx:3]: Done 👍

**Review by triggertoo — APPROVED** [review]

**Pull request merged**